portfolio.html

```diff
@@ -29,7 +29,7 @@
     
         
     
```

Comment:
- [ ] On the logo `welcome` please change this from a div to an anchor tag `<a>`
This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

- [ ] Also your there is no hamburger icon. This icon needs to be present for the mobile design

<img width="380" alt="Screenshot 2022-10-01 at 18 04 26" src="https://user-images.githubusercontent.com/48546992/193415814-e381bf9f-5d0b-4458-b307-f9bc5db18379.png">